extensions/wikia/Email/Email.setup.php

```diff
@@ -111,5 +111,9 @@
 $wgGroupPermissions['helper']['access-sendemail'] = true;
 $wgGroupPermissions['translator']['access-sendemail'] = true;
 
-$wgAddGroups['staff'][] = 'translator';
-$wgRemoveGroups['staff'][] = 'translator';
+if ( is_array( $wgAddGroups['staff'] ) ) {
+	$wgAddGroups[ 'staff' ][] = 'translator';
+}
+if ( is_array( $wgRemoveGroups[ 'staff' ] ) ) {
+	$wgRemoveGroups[ 'staff' ][] = 'translator';
+}
```

Comment:
This emits warnings in `dev` since these are set to booleans rather than arrays in `dev`.